### PR TITLE
Disable gradient tape profiling tests

### PR DESCRIPTION
**Pull request by adimux**
### Description of changes:
Tests related to profiling using gradient tape are failing
and blocking the TF2.7 release.
Since this feature has never been released, we can safely
disable the tests, until we make a decision about supporting this
feature.

#### Style and formatting:

I have run `pre-commit install && pre-commit run --all-files` to ensure that auto-formatting happens with every commit.

#### Issue number, if available

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tests/profiler/tensorflow2/test_native_tf2_profiler.py [hunk 1]
```diff
@@ -146,7 +146,10 @@ def _verify_timeline_files(out_dir):
             "process_sort_index",
         }
 
-
+# Skipping because the tests are failing.
+# Support for profiling using gradient tape has never been released publicly
+# and since we're planning on deprecating profiler v1, we can just disable the tests
+@pytest.mark.skip(reason="unblock TF2.7 release")
```
Comment (ndodda-amazon) [inline tests/profiler/tensorflow2/test_native_tf2_profiler.py:152]: The tests only fail for TF 2.7, right? Then let's use `skipif` to skip the test for all TF versions 2.7 and above.

**Review by ndodda-amazon — CHANGES_REQUESTED** [review]
If profiler_v1 is deprecated, I would actually just recommend removing [this line](https://github.com/awslabs/sagemaker-debugger/blob/master/config/tests.sh#L37). So you don't need to manually skip each test in this file (and you can ignore the other comment I made).

**Comment by adimux** [comment]
xgboost tests are failing in the nightly build of `sagemaker-debugger`, so this PR is ok to merge

**Review by ndodda-amazon — APPROVED** [review]
LGTM, great work!

**Review by josephevans — APPROVED** [review]
Thanks, LGTM.

File: tests/profiler/tensorflow2/test_native_tf2_profiler.py (hunk 1 above)
Comment (hsl89) [reply tests/profiler/tensorflow2/test_native_tf2_profiler.py]: Hi @ndodda-amazon, do tests under `tests/profiler/tensorflow2/` depends on the code changes applied to the tf code base?

**Comment by Roshrini** [comment]
LGTM. Thanks

**Review by Roshrini — APPROVED** [review]
